config-nyc-pt.yml

```diff
@@ -0,0 +1,19 @@
+graphhopper:
```

Comment:
what's this file used for?

Reply:
This is the GH config file, used to specify settings for GH/location of OSM+GTFS files/which vehicle profiles we want. The way GH knows that it should build graphs for transit requests and not auto/bike/ped is whether or not the `gtfs.file` line is included in the config (and in this case, you can't include the usual vehicles that our standard configs use), so I made a special config for the NYC region for testing here.

In a productionized setting, we'll want to come up with templating for configs like this (you can provide each of these lines as command line args, so we can make a general PT config that we inject GTFS paths/etc into)